templates/ecommerce/checkout.php

```diff
@@ -66,69 +67,64 @@
 						<div class="tutor-billing-fields">
 							<?php require tutor()->path . 'templates/ecommerce/billing-form-fields.php'; ?>
 						</div>
-
-						<h5 class="tutor-fs-5 tutor-fw-medium tutor-color-black tutor-mb-24 tutor-mt-20">
-							<?php esc_html_e( 'Payment Method', 'tutor' ); ?>
-						</h5>
-						<div class="tutor-checkout-payment-options tutor-mb-24">
-							<input type="hidden" name="payment_type">
-							<?php
-							$payment_gateways = tutor_get_all_active_payment_gateways();
-							if ( empty( $payment_gateways ) ) {
-								?>
-								<div class="tutor-alert tutor-warning">
-									<?php esc_html_e( 'No payment method has been configured. Please contact the site administrator.', 'tutor' ); ?>
-								</div>
+						<div class="tutor-payment-method-wrapper">
+						<?php if ( ! $is_zero_price ) : ?>
```

Comment:
Here we have added a filter to show and hide payment method.
Check PR https://github.com/themeum/tutor/pull/1661/files#diff-a622ef087badd5c86bdfa938cd4d67ed1977b2f684218158db13770bdbf984df